theme/beevibrant/templates/page.html

```diff
@@ -11,7 +11,7 @@
 {% block content %}
 {{ super() }}
 {% if page %}
-<section class="tg-sectionspace tg-haslayout" id="euroscipy-page">
```

Comment:
This one as well.

Reply:
this one is a consequence of adding the schedule back, but it actually fixed rendering of the most pages on my mobile, before that I couldn't see titles on some of the pages because they were covered by the blue stripe on the top